pkg/leader/leader.go

```diff
@@ -109,7 +109,33 @@ func Become(ctx context.Context, lockName string) error {
 			log.Info("Became the leader.")
 			return nil
 		case apierrors.IsAlreadyExists(err):
-			log.Info("Not the leader. Waiting.")
+			existing := &corev1.ConfigMap{}
+			key := crclient.ObjectKey{Namespace: ns, Name: lockName}
+			err := client.Get(ctx, key, existing)
```

Comment:
Do we need to repeat this `client.Get()` call for the lock config map? Wondering if we have a valid `existing` variable still from when we got it in lines 75-77?

Reply:
Yes `existing` still seems to be valid from above, since we're not using any fields that would be updated after attempting to `Create(cm)`.

Reply:
This might actually need to be refreshed (see #3059).